sources/manager_wakatime.py

```
@@ -0,0 +1,127 @@
+                return None
+
+            data = response.json()
+
+            if "data" not in data:
+                DBM.w("WakaTime response missing 'data' field")
+                return None
+
+            DBM.g("WakaTime stats fetched successfully!")
+            return data["data"]
+
+        except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)